docs: Added explore Traces plugin to Tempo Quick start tutorial (#4377)
grafana/tempo: docs/sources/tempo/getting-started/docker-example.md

    
<!-- INTERACTIVE page step2.md END -->

<!-- INTERACTIVE page step3.md START -->

> ## Explore Traces Plugin

Reviewer: Using sentence case
```suggestion
## Explore Traces plugin
```

> Another method to explore traces within Grafana is via the [Explore Traces](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/) plugin. This plugin offers an opinionated non query-based approach to exploring traces. Lets take a look at some of its key features and panels. 

Reviewer: Any sugestions for a different word besides "explore traces using Explore Traces"
```suggestion
Another method to explore traces within Grafana is using the [Explore Traces](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/) plugin. This plugin offers an opinionated non query-based approach to exploring traces. Lets take a look at some of its key features and panels. 
```

1. Open a browser and navigate to [http://localhost:3000/a/grafana-exploretraces-app](http://localhost:3000/a/grafana-exploretraces-app).

> 2. Within the filter bar there is a dropdown menu set to `Rate` of `Full traces`. Change this to `Duration` and `All spans`.

Reviewer: ```suggestion
2. Within the filter bar, there is a dropdown menu set to **Rate** of **Full traces**. Change this to **Duration** and **All spans**.
```

> This should provide an updated panel view like this.

Reviewer: ```suggestion
This updated panel view looks like this:
```

{{< figure align="center" src="/media/docs/tempo/explore-spans-error-view.png" alt="Explore Traces panel" >}}

Breakdown of the view:

> * The histogram at the top shows the distribution of span durations. The lighter the color, the more spans in that duration bucket. In this example most spans fall within `537ms` which is considered the average duration for the system.

Reviewer: ```suggestion
* The histogram at the top shows the distribution of span durations. The lighter the color, the more spans in that duration bucket. In this example, most spans fall within `537ms`, which is considered the average duration for the system.
```

> * There are some high peaks in the histogram, which indicate spans that are taking longer than the average (As high as `2.15s`). These are likely to be the spans that are causing performance issues. Lets investigate further to see if we can identify the root cause.

Reviewer: ```suggestion
* The high peaks in the histogram indicate spans that are taking longer than the average (As high as `2.15s`). These are likely to be the spans that are causing performance issues. Let's investigate further to see if we can identify the root cause.
```

Select `Slow traces` tab in the navigation bar to view the slowest traces in the system. 

{{< figure align="center" src="/media/docs/tempo/slow-trace-view.png" alt="Slow traces panel" >}}

`shop-backend` appears to be the primary culprit for the slow traces. This happens when a user initiates the `article-to-cart` operation. From here we can select the Trace Name which will open the Trace View panel. 

{{< figure align="center" src="/media/docs/tempo/slow-trace-trace-view.png" alt="Trace View panel" >}}

The Trace View panel provides a detailed view of the trace. The panel is divided into three sections:
* The top section shows the trace ID, duration, and the service that generated the trace.
* The middle section shows the trace timeline. Each span is represented as a horizontal bar. The color of the bar represents the span's status. The width of the bar represents the duration of the span.
* The bottom section shows the details of the selected span. This includes the span name, duration, and tags.

> Drilling into the `shop-backend` span, we can see that the `place-articles` operation has an exception event tied to it. This is likely the root cause of the slow trace. 

Reviewer: ```suggestion
Drilling into the `shop-backend` span, you can see that the `place-articles` operation has an exception event tied to it. This is likely the root cause of the slow trace. 
```
